CONTRIBUTING.md

```diff
@@ -0,0 +1,69 @@
+# How to contribute
```

Comment:
Move it for https://github.com/operator-framework/operator-lifecycle-manager/pull/1438/files